freight/notifiers/generic.py

```diff
@@ -0,0 +1,46 @@
+__all__ = ["GenericNotifier"]
+
+from freight import http
+from freight.models import App, Task, TaskStatus, User
+
+from .base import Notifier, NotifierEvent, generate_event_title
+
+
+class GenericNotifier(Notifier):
+    def get_options(self):
+        return {"webhook_url": {"required": True}}
```

Comment:
Would you wanna expose configuration to add extra headers to the request? That's tpyically a thing you might wanna do, say, for an API key or some other authentication needed.

Reply:
@mattrobenolt would a secret key be sufficient or would we want a signature to validate the payload?

Reply:
I'm just suggesting a configurable dict/list of headers, then they can be whatever.